src/home_content.json

```diff
@@ -0,0 +1,144 @@
+				},
+				{  
+					"par": "Our code of conduct is simple- 'Be nice,Be respectful' which is the one of the many reasons the community coexists in harmony .The members should contribute actively keeping the policies of being repectful to everyone and refrainging from conflicts, unruly behaviour and speech.Anyone who wishes to join the community is expected to follow these guidelines.\n If you are subject to or witness unacceptable behavior, or have any other concerns,please notify a community organizer or adminas soon as possible."
+			
+				}
+			]
+		},
+	    {
+			"title": "Ways To Contribute",
+			"content":[
+				{
+					"par": "You can start contributing to our various open source projects by-1)Creating issues on Github regarding the project 2)Debugging the issues 3)Creating pull requests, testing pull requests 4)Create mockups for new applications or redesign the current features 5)Research about UI/UX improvements and accessibilty features. 6)Create app walkthroughs with documents or videos,UI testing once the PR is submitted for the UI issue"
+
+				},
+				{
+					"par": "Each active repository has a stream to direct questions.Issues labeled as 'First Timers Only' are meant for contributors who have not contributed to the project yet.Please choose other issues to contribute to, if you have already contributed to these type of issues.Make sure to follow the Commit Message Style Guide when submitting PRs which will require review by atleast one maintainer to be merged to the main code."
+				}
+			]
+		},
+		{
+			"title": "Our Projects",
+			"content":[
+				{
+					"par": "First and foremost,choose a programming language of your choice.Once you are done with selecting a programming language search for a project .Here's a list of our open source projects - "
+				},
+				{
+					"par": "1)Mentorship System \n Mentorship System is an application that matches women in tech to mentor each other, on career development, through  relations during a certain period of time. \n It is an open source project proposed for Google Summer of Code 2018 with AnitaB.org Open Source,an idea that consists of a mobile application that matches women to help and support both parties. One is the mentor that will be a support/accountability buddy and or give advice to a woman that wants mentorship, for a period of time established by both parties.This would not focus only on students, this would target women from any age, background, work in tech experience, and so on.This project uses Kotlin for frontend and Python for backend. \n Flutter and iOS versions are also available."
```

Comment:
please replace `1)Mentorship` with `1) Mentorship`, there should be no space after `\n`